modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,72 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+== Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily in order to clean up any stranded IP allocations that may result in exhausting IPs and hence prevent new Pods from getting an IP allocated to them.
+
+The following procedure shows how to change the frequency at which the IP reconciler runs. 
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Run the following command to create a `ConfigMap` named `whereabouts-config` in the `openshift-multus` namespace with a specific cron expression for the IP reconciler:
++
+[source,terminal]
+----
+$ oc create configmap whereabouts-config -n openshift-multus --from-literal=reconciler_cron_expression="*/15 * * * *"
+----
++
+This cron expression indicates the IP reconciler should run every 15 minutes. Adjust the expression based on your specific requirements. 
```

Comment:
```suggestion
This cron expression indicates the IP reconciler runs every 15 minutes. Adjust the expression based on your specific requirements. 
```
Does it run reliably every 15 minutes or not? Maybe add something about if it doesn't run on schedule if there is a chance it won't?